src/admin-views/aggregator/fields/schedule.php

```diff
@@ -40,7 +40,8 @@ class="tribe-dependent"
 				type="hidden"
 				name="aggregator[<?php echo esc_attr( $origin ) ?>][schedule_day]"
 				id="tribe-ea-field-<?php echo esc_attr( $origin ) ?>_schedule_day"
-				class="tribe-ea-field tribe-ea-dropdown tribe-ea-size-small"
+				class="tribe-ea-field tribe-ea-dropdown tribe-ea-size-medium"
+				style="margin-left: 0;"
```

Comment:
I see a few of these inline styles. Do those _have_ to be inline (and if so, I'd love to know why). If they don't need to be inline, can you move them to a `.pcss` file?